README.md

```diff
@@ -25,7 +25,7 @@ It can be used to convert received tuples to objects via `crud.unflatten_rows` f
 For example:
 
 ```lua
-res, err = crud.select('customers')
+res, err = crud.select('customers', nil, {fullscan = true})
```

Comment:
As part of this activity we had intention to get rid of unlimited selects in the documentation. I would consider crud examples as part of this activity. It is not nice to suggest `fullscan = true` in the first example.

Reply:
Sure. It has been replaced with a condition.

Similar actions were done in the doc/select.md. There is an unlimited select, but I think it is logical from the context:
```patch
--- a/doc/select.md
+++ b/doc/select.md
@@ -60,11 +60,11 @@ Let's check ``developers`` space content to make other examples more clear. Just
 **Example:**
 
 ```lua
-crud.select('developers', nil, { first = 6 })
+crud.select('developers', nil, { first = 6, fullscan = true })
```
Is it better to replace it with a condition?

Reply:
Before I read the resulting documentation, I had thought that the proposed approach may be helpful in some cases despite many design flaws. Now I see that it confuses more than helps.

We have forward and backward pagination examples. All select calls are limited, but they need either `fullscan = true` or `{'>', 'id', 0}` condition (which is obviously just to get rid of the warning, because it is evaluated to true for all tuples). It has no sense.

But first example from the 'Avoiding full scan` section, which performs full scan, will not give the warning. It has no sense too.

Ugh. I reached product managers once again. I don't know, how to review the implementation, if the proposal itself has so many flaws.

Please, don't take it as criticism of your work. It is mine fault: ideally I should spot all those points before asking a developer to start working on the implementation.

Reply:
requirements have been updated:
https://github.com/tarantool/tarantool/issues/7129